.eslintignore

```diff
@@ -1,4 +1,2 @@
-coverage
-generators/**/templates
-node_modules
-travis
+**/templates/
+**/resources/
```

Comment:
Shouldn't this still include `node_modules`? Or is it excluded by default?

Reply:
By default